src/rpc/claimtrie.cpp

```diff
@@ -398,17 +398,16 @@ UniValue getclaimbyid(const UniValue& params, bool fHelp)
     LOCK(cs_main);
     uint160 claimId = ParseClaimtrieId(params[0], "Claim-id (parameter 1)");
     UniValue claim(UniValue::VOBJ);
-    std::string name;
     CClaimValue claimValue;
+    std::string name, sValue;
     pclaimTrie->getClaimById(claimId, name, claimValue);
-    if (claimValue.claimId == claimId)
+    if (pclaimTrie->getClaimById(claimId, name, claimValue) && getValueForClaim(claimValue.outPoint, sValue))
```

Comment:
`getClaimById` should not be called twice.

Reply:
thanks, squashed and force pushed